stglib/vec/dat2cdf.py

```diff
@@ -34,21 +36,19 @@ def dat_to_cdf(metadata):
 
     ds = aqdutils.check_attrs(ds, inst_type="VEC")
 
-    dsvhd = load_vhd(basefile)
+    dssen = load_sen(basefile, metadata)
 
-    dssen = load_sen(basefile)
-
-    r = np.shape(dssen.Heading)[0]
-    senburstlen = int(ds.attrs["VECSamplesPerBurst"] / ds.attrs["VECSamplingRate"] + 1)
+    # r = np.shape(dssen.Heading)[0]
+    # senburstlen = int(ds.attrs["VECSamplesPerBurst"] / ds.attrs["VECSamplingRate"] + 1)
     ds.attrs["sample_interval"] = 1 / ds.attrs["VECSamplingRate"]
 
-    mod = r % senburstlen
-    if mod:
-        print(f"{ds.attrs['VECSamplesPerBurst']=}")
-        print(f"{ds.attrs['VECSamplingRate']=}")
-        histtext = "Number of rows in .sen file is not a multiple of ds.attrs['VECSamplesPerBurst']/ds.attrs['VECSamplingRate'] + 1; truncating to last full burst"
-        ds = utils.insert_history(ds, histtext)
-        dssen = dssen.sel(time=dssen.time[0:-mod])
+    # mod = r % senburstlen
```

Comment:
Why is this code commented out? Will this still work with burst mode data that has in incomplete final burst? (This can happen when the instrument is stopped in the middle of collecting a burst.)

Reply:
I have this commented out because I don't shape the data into bursts until the cdf2nc step using MultiIndex. And yes, I just modified it so it can handle an incomplete final burst. If the deploy/recovery times do not trim it off, then the burst will still be presented in the .nc file but will just not be complete. 

I did this a little differently because there could be incomplete bursts in the middle of one of the raw data files. Or a skipped sample (which I noticed happening in a .sen file). So I configure the time using a combination of the .dat file shape, .dat file burst number (unless it is continuous), and the time from the .vhd file. This all happens in load_dat from dat2cdf.py.